js/utils.js

```diff
@@ -1511,6 +1511,51 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
     });
   }
 
+  function setFilterValues(data) {
+    return new Promise(function(resolve) {
+      if (!data.filterOptions.length) {
```

Comment:
In case the filter options don't exist.

```suggestion
      if (!(data.filterOptions || []).length) {
```